src/feedback/templates/feedbackTemplates.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export class FeedbackTemplates {
```

Comment:
Should we localize the view, or start as English only?

Reply:
The other web views also don't localize. As it stands, we would either need to import the localization module (none of the views import anything other than vue) or inject all the strings. Doesn't really seem worth it at this point.

Reply:
Ok. When this PR is merged, could you please add an issue into the GitHub repo to localize this view? We should start tracking these types of things (I'm thinking per occasion).

Reply:
Will do. I think we should take care of this if and when we change the web views